src/gql/fragments/patchesPage.ts

```diff
@@ -0,0 +1,26 @@
+import { gql } from "@apollo/client";
+
+/* eslint-disable */
```

Comment:
graphql/template-strings thinks the fragment defined in this file is never used, you can read about the issue [here](https://github.com/apollographql/eslint-plugin-graphql/issues/215). 

I think a better work around could be to disable graphql/template-strings on this entire file. The alternative solution is to create a validator safelist and be sure to exclude the NoUnusedVariables validator which is [recognized as clunky](https://github.com/apollographql/eslint-plugin-graphql/issues/98) since our safelist will become outdated as soon as new validators are released.

Reply:
That makes sense can you disable the `graphql/template-strings` rule specifically then. So we don't completely lose eslint validation on this file.

Reply:
@khelif96 specifically disabling graphql/template-strings f[ails the compile step](https://evergreen.mongodb.com/task_log_raw/spruce_ubuntu1604_compile_patch_2041a5750f7ffed7de3cf68ae09c52c998ce8272_5fda7ef47742ae0e4f42290f_20_12_16_21_41_09/0?type=T#L125) however eslint-fix does not report any errors